pkg/graveler/committed/merge.go

```diff
@@ -138,7 +155,7 @@ func (m *merger) sourceBeforeDest(sourceValue *graveler.ValueRecord) error {
 }
 
 // handleAll handles the case where only one Iterator from source or dest remains
-func (m *merger) handleAll(iter Iterator) error {
+func (m *merger) handleAll(iter Iterator, strategyToInclude graveler.MergeStrategy) error {
```

Comment:
I like the way you handled the conflict, add to the documentation of the function the difference between merge strategy and the strategy to include so it won't be confusing

Reply:
Documentation added. It was definitely needed - thanks for that comment